From 25202045186262a081c960461a8b791f84fccb5c Mon Sep 17 00:00:00 2001
From: Carl Schwan <carl@carlschwan.eu>
Date: Sun, 22 Jun 2025 23:32:57 +0200
Subject: [PATCH] Do not request password twice

Otherwise the second time will crash as the password was already emitted
and the job deleted
---
 resources/imap/settingspasswordrequester.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/resources/imap/settingspasswordrequester.cpp b/resources/imap/settingspasswordrequester.cpp
index 0892a9cba..9de998c98 100644
--- a/resources/imap/settingspasswordrequester.cpp
+++ b/resources/imap/settingspasswordrequester.cpp
@@ -57,7 +57,6 @@ void SettingsPasswordRequester::requestPassword(RequestType request, const QStri
                 m_readPasswordJobs.removeAll(job);
             });
             m_readPasswordJobs << job;
-            job->start();
         } else {
             auto password = m_resource->settings()->password();
             if (password.isEmpty()) {
-- 
2.50.0

